docs/howto/cylc-ui.md

```diff
@@ -1,60 +1,113 @@
+   node --version
+   ```
+   For comparison devs have used node at 10.13.x and 12.x.x.
 
 3. Install
    [cylc/cylc-flow](https://github.com/cylc/cylc-flow/) and
    [cylc/cylc-uiserver](https://github.com/cylc/cylc-uiserver/):
-  ```bash
-  pip install -e "${DEVDIR}/cylc-flow"[all]
-  pip install -e "${DEVDIR}/cylc-uiserver"[all]
-  ```
-  where `${DEVDIR}/cylc-flow` and `${DEVDIR}/cylc-uiserver` are the locations
-  of your local (git) worktrees of the two projects.
+
+   ```bash
+   pip install -e "${DEVDIR}/cylc-flow"[all]
+   pip install -e "${DEVDIR}/cylc-uiserver"[all]
+   ```
+
+   Where `${DEVDIR}/cylc-flow` and `${DEVDIR}/cylc-uiserver` are the locations
+   of your local (git) worktrees of the two projects.
 
 4. Install [cylc/cylc-ui](https://github.com/cylc/cylc-ui/).
-  ```bash
-  cd $DEVDIR/cylc-ui
-  npm install --prefix="${DEVDIR}/cylc-ui"
-  npm install -g configurable-http-proxy
-  ```
-  where `${DEVDIR}/cylc-ui` is the location
-  of your local (git) worktree of the project.
-  
+
+   > **Note:** We prefer `yarn`.
+
+   ```bash
+   cd $DEVDIR/cylc-ui
+   yarn install
+   ```
+
+   Where `${DEVDIR}/cylc-ui` is the location
+   of your local (git) worktree of the project.
 
 5. Run `jupyterhub`.
-  ```bash
-  conda activate my-example-env
-  jupyterhub -f ${DEVDIR}/cylc-uiserver/jupyterhub_config.py
-  ```
+
+   ```bash
+   conda activate my-example-env
+   jupyterhub -f ${DEVDIR}/cylc-uiserver/jupyterhub_config.py
+   ```
 
 6. Run UI.
-  ```bash
-  conda activate my-example-env
-  npm run --prefix="${DEVDIR}/cylc-ui" build:watch
-  ```
+   ```bash
+   conda activate my-example-env
+   yarn run build:watch
+   ```
+
+7. Open a browser and navigate to `http://localhost:8000/`.
 
-7. Open a browser and navigate to http://localhost:8000/.
    You will be asked to log in with your desktop credentials if you have not
    done so before.
-  
 
+## Running Your First Workflow
+
+1. Create a basic Cylc workflow:
+
+   ```bash
+   mkdir -p ~/cylc-run/foo
+   cat > ~/cylc-run/foo/flow.cylc <<__HERE__
+   [scheduling]
+       # a cycling workflow where the cycles are numbered as integers
+       cycling mode = integer
+       initial cycle point = 1
+       [[graph]]
+           # tasks which run in each cycle until you stop the workflow
+           P1 = """
+               # a => b means "b should wait until a has run"
+               # [-P1] means "from the previous cycle"
+               b[-P1] => a => b => c
+           """
+   __HERE__
+   ```
+
+2. Run the workflow:
+
+   ```bash
+   conda activate my-example-env
+   cylc run foo
+   ```
+
+3. Watch it run on the CLI:
+
+   ```bash
+   cylc tui foo
+   ```
+
+4. Watch it run in the UI:
 
+   Fire up jupyterhub as before, open a browser and navigate to
+   `http://localhost:8000/`.
 
+5. Stop it:
 
+   (otherwise it will just keep running forever)
 
-*It is important that jupyterhub is not installed on conda after nodejs, or it
-will fatally downgrade nodejs.
```

Comment:
Looks like this is no longer the case - https://github.com/conda-forge/jupyterhub-feedstock/blob/master/recipe/meta.yaml#L81

Reply:
:+1: wasn't aware of that. Good to know!